frontend/src/components/modal/modal.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'; 
+import ReactDOM from 'react-dom'; 
+import styled from 'styled-components';
+
+const ModalOverlay = styled.div`
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 0;
+    width: 100vw;
+    height: 100vh;
+    background-color: #000;
+    opacity: .5;
+`
+const ModalWrapper = styled.div`
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 100;
+    width: 100%;
+    height: 100%;
+    display: flex; 
+    justify-content: center; 
+`
+
+const ModalEle = styled.div`
+    border: 1px solid; 
+    background: white; 
+    position: relative;
+    border-radius: 3px;
+    top: 37px; 
+    width: 550px;
+    height: 200px; 
+    display: flex; 
+    justify-content: center; 
+    flex-direction: column;
+    align-items: center; 
+    @media (max-width: 420px) {
+        height: 300px; 
+    }
+    
+`
+const ModalHeader = styled.div`
+    position: relative; 
+    top: -22px; 
```

Comment:
Don't know if the `top` property is going to work with `position: relative;`... consider using negative margin if you need to pull the div up.